path.go

```diff
@@ -14,12 +14,8 @@
 
 package zetcd
 
-import (
-	"path"
-)
-
 func mkPath(zkPath string) string {
-	p := path.Clean(zkPath)
```

Comment:
`mkPath` itself shouldn't call `path.Clean` because mkPath can't fail (and allows a number of disallowed paths through by forcing them to be correct enough for Zookeeper, while still not matching them).

i.e. Zookeeper doesn't allow `/something/../something2` as a keyname, but an application trying to do this should get an error that this is wrong, not have it's path silently changed to `/something2`.

I'm thinking `mkPath` should implement pure ZK -> etcd mapping, and have a utility function which checks for naming consistency (according to the docs: https://zookeeper.apache.org/doc/r3.1.2/zookeeperProgrammers.html#ch_zkDataModel) which can fail to sanitize user input.

I agree with @heyitsanthony that the FlagSequence code probably needs to be handled in Create since it seems special-casey even for ZK.